### DISCUSSION TITLE
Use custom/default telescope picker for `vim.ui.select`

### DISCUSSION THREAD

**Discussion by Subjective** [stevearc/dressing.nvim#108] [Q&A] [answered]
Is there any way to use the default telescope picker for `vim.ui.select`? I've altered the hl groups for telescope to get something similar to what Nvchad, but because there is no spacing between the telescope prompt and selection window, the prompt's proportions are off. I looked into the different telescope themes, but those don't seem to alter the window layout.

Ideally, I would like to get the same kind of picker used in `:Telescope find_files`, but with the height and width scaled down and preview disabled:

<img width="763" alt="image" src="https://github.com/stevearc/dressing.nvim/assets/56745535/f5a1333a-75b7-4f7c-ad48-e444a90f2d31">

This is what it looks like right now:
<img width="815" alt="image" src="https://github.com/stevearc/dressing.nvim/assets/56745535/2fd44fb5-5408-481f-b77f-443b6108721c">

Thanks!

**Comment by stevearc** [comment]
As mentioned in the docs, you can put whatever you want in the `select.telescope` option and it will get passed in directly to the telescope picker. Any possible telescope customization should be available via that method.

As for how to actually do what you're asking, I'm not sure. I don't personally do any telescope UI tweaking.

**Comment by Subjective** [comment, marked as answer]
For future reference, I was able to get the style I wanted with these options:

```lua
opts = {
  select = {
    backend = { "telescope"  },
    telescope = {
      layout_strategy = "horizontal",
      layout_config = {
        preview_cutoff = false,
        width = function(_, max_columns, _) return math.min(max_columns, 80) end,
        height = function(_, _, max_lines) return math.min(max_lines, 15) end,
      },
    },
  },
},
```

<img width="881" alt="image" src="https://github.com/stevearc/dressing.nvim/assets/56745535/383afc68-b773-4d75-9d1e-95e7a259dd23">
